numpy-stubs/__init__.pyi

```diff
@@ -778,3 +779,17 @@ trunc: ufunc
 
 # TODO(shoyer): remove when the full numpy namespace is defined
 def __getattr__(name: str) -> Any: ...
+
+# Warnings
+warnings = _warnings
```

Comment:
Har, I think we should leave out warnings here and declare that any code using `np.warnings` is officially broken.